src/scripts/main.js

```diff
@@ -1,3 +1,336 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const table = document.querySelector('table');
+const th = document.querySelectorAll('th');
+let previous = '';
+
+for (const cell of th) {
+  cell.setAttribute('data-Asc', true);
+}
+
+function toNumber(string) {
+  const number = string.slice(1).split('').filter(el => el !== ',').join('');
+
+  return +number;
+}
+
+function stringCompare(a, b, child) {
+  const result
+  = a.children[child].textContent.localeCompare(b.children[child].textContent);
+
+  return result;
+}
+
+// Sorting function
+
+const sort = (ev) => {
+  const tr = document.querySelectorAll('tr');
+  const tableContent = [...tr].slice(1, tr.length - 1);
+
+  tbody.innerHTML = '';
+
+  if (ev.target.dataset.asc === 'true'
+  || (ev.target.dataset.asc === 'false' && previous !== ev.target.innerText)) {
```

Comment:
Also move ev.target.dataset.asc to a variable
```suggestion
  if (ev.target.dataset.asc
  || (!ev.target.dataset.asc && previous !== ev.target.innerText)) {
```